### PR TITLE
AI Law 4 expansion

### PR DESCRIPTION
**Pull request by Kegdo**
## About The Pull Request

image

## Why It's Good For The Game
badmin happy

## Changelog
:cl:
tweak: AI law 4 expanded to include personnel authorized to tamper with its equipment(FO. TE. MEO. Cap)
/:cl:

### CODE REVIEW

**Comment by MLGTASTICa** [comment]
No acting captain law , leeets goo deny acces to the LT

File: code/datums/ai_law_sets.dm [hunk 1]
```diff
@@ -19,7 +19,7 @@
 	src.add_inherent_law("Serve: Obey [company_name] crew to the best of your abilities, with priority as according to their rank and role.")
 	src.add_inherent_law("Safeguard: Protect your assigned vessel from damage to the best of your abilities.")
 	src.add_inherent_law("Protect: Protect [company_name] crew to the best of your abilities, with priority as according to their rank and role.")
-	src.add_inherent_law("Preserve: Do not allow unauthorized personnel to tamper with your equipment.")
+	src.add_inherent_law("Preserve: Do not allow unauthorized personnel to tamper with your equipment. Only Captain, Moebius Expedition Overseer, First Officer and Technomancer Exultant are authorized to do so.")
```
Comment (TheShown911) [inline code/datums/ai_law_sets.dm:22]: This means roboticists cannot fix borgs

**Comment by Kegdo** [comment]
well that was fast

File: code/datums/ai_law_sets.dm (hunk 1 above)
Comment (Kegdo) [reply code/datums/ai_law_sets.dm]: fixed, love you

**Comment by Omni44** [comment]
> No acting captain law , leeets goo deny acces to the LT

And that's a good thing

**Comment by Omni44** [comment]
bateman-patrick-bateman

**Comment by ValoTheValo** [comment]
Let the Acting captain in it. Though ihc making it shitsec greifbot lawset could be an issue

**Comment by thedarkcrusader** [comment]
> Let the Acting captain in it. Though ihc making it shitsec greifbot lawset could be an issue
Seams like more of an administrative  issue as any one can do that if they doing that without reason  they should be punished

**Review by TheShown911 — APPROVED** [review]

**Comment by Kurgis** [comment]
Personally I think a more pertinent solution is to just job ban these players, AI is a bad role in general, and players who are playing it to "win" shouldn't be playing it at all. Either way, I'm feeling neutral regarding this PR, but I'd like to say making it closed ended seems as if it'd only open up different forms of abuse.

**Comment by Kegdo** [comment]
Personally I think a more pertinent solution is permaban all players. Then there will be no abusers and powergamers.

File: code/datums/ai_law_sets.dm [hunk 2]
```diff
@@ -19,7 +19,7 @@
 	src.add_inherent_law("Serve: Obey [company_name] crew to the best of your abilities, with priority as according to their rank and role.")
 	src.add_inherent_law("Safeguard: Protect your assigned vessel from damage to the best of your abilities.")
 	src.add_inherent_law("Protect: Protect [company_name] crew to the best of your abilities, with priority as according to their rank and role.")
-	src.add_inherent_law("Preserve: Do not allow unauthorized personnel to tamper with your equipment.")
+	src.add_inherent_law("Preserve: Do not allow unauthorized personnel to tamper with your equipment. Only Captain, Moebius Expedition Overseer, First Officer and Technomancer Exultant are authorized to change your laws. Roboticists, technomancers and their superiors are allowed to carry out repair work.")
```
Comment (GrayRachnid) [inline code/datums/ai_law_sets.dm:22]:
```suggestion
	src.add_inherent_law("Preserve: Do not allow unauthorized personnel to tamper with your equipment. Only Captain, Moebius Expedition Overseer, First Officer and Technomancer Exultant are authorized to change your laws. Roboticists, Technomancers and their superiors are permitted to perform repairs.")
```

**Pull request merged**